src/components/ErrorMessage/ErrorMessage.tsx

```
@@ -0,0 +1,24 @@
+import classNames from 'classnames';
+import { ErrorType } from '../../types/ErrorType';
+import { getMessage } from '../../services/errorServices';
+
+type ErrorProps = {
+  error: ErrorType;
+};
+
+const ErrorMessage: React.FC<ErrorProps> = ({ error }) => {
+  return (
+    <div
+      data-cy="ErrorNotification"
+      className={classNames(
+        'notification is-danger is-light has-text-weight-normal',
+        { hidden: !error.isVisible },
+      )}
+    >
+      <button data-cy="HideErrorButton" type="button" className="delete" />
```

The close button (`HideErrorButton`) should have an `onClick` handler to hide the error message. Currently, it's missing the logic to update the `isVisible` state in the parent component to hide the notification.